credible/faq.mdx

```diff
@@ -148,10 +140,11 @@ No. We have performed worst-case stress tests, using the most inefficient operat
 
 ### How do you extend the validation logic of the sequencer to the dApp?
 
-Flashbots introduced a piece of software that enables a Sequencer to have an external block-builder that implements custom block-building logic, without changing the sequencer itself.
-This is also what powers Flashblocks. 
+For OP stack chains, Flashbots introduced a piece of software that enables a Sequencer to have an external block-builder that implements custom block-building logic, without changing the sequencer itself. This is also what powers Flashblocks. 
```

Comment:
Secuencer is capitalized the first mention in this sentence but not in the second. Is this on purpose?